calico/felix/fetcd.py

```diff
@@ -221,10 +222,12 @@ def load_initial_dump(self):
         for child in initial_dump.children:
             profile_id, rules = parse_if_rules(child)
             if profile_id:
+                profile_id = intern(profile_id.encode("utf8"))
```

Comment:
Nit picky really, but it would be cleaner if the interning was done in `parse_if_blah` for all `blah`.